.gitignore

```diff
@@ -4,3 +4,4 @@
 node_modules
 build
 dist
+play.d2lang.com
```

Comment:
what's this

Reply:
ci.sh builds the binary.

This is from when the module name was different, will update